### DISCUSSION TITLE
Symfony profiler not showing up+mixed content error

### DISCUSSION THREAD

**Discussion by daphneaugier** [symfony/symfony#45019] [Q&A]
Hello everyone,

My Symfony profiler (v5.3.12) is really bugging me because it is not showing up/sending errors in the console.

I have four different environments in my project but I only need to see it on `dev` and `stage`. 
On `dev` the profiler was not showing up and in the console I could see this error `Mixed Content: The page at '***' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint '***'. This request has been blocked; the content must be served over HTTPS.` 
I was able to make it appear again by putting in the **web_profiler.yaml** file `web_profiler: intercept_redirects` to `true`. This fix was also removing the Mixed Content error in the console.
However, in my `stage` environment, this fix was only removing the console error but the profiler is still not showing up. I tried adding `APP_DEBUG `to `true` in the stage environment but it doesn't do anything.
I would like to know why is the Mixed Content error related to the Symfony Profiler and how I can make the profiler show up in the stage environment? Is the profiler fragile? Is it related to the version of Symfony that I am using?

Thank you for the help!

**Comment by stof** [comment]
I think this might happen if your code runs being a TLS-terminating proxy that is not properly configured as a trusted proxy, as Symfony would then consider that your page is loaded with HTTP rather than HTTPS and will generate absolute URLs using `http://`. And the profiler toolbar relies on some absolute URLs to be loaded (which is necessary for the feature of replacing the toolbar for ajax calls, as the ajax call might be to a different domain than the current page).

**Reply by nlipkanov** [reply]
Just a follow up question: Is this the solution for of what you @stof are talking about: https://symfony.com/doc/current/deployment/proxies.html

I have the same issue now (Symfony v6.1, Symfony Debug Bundle v6.1).

**Reply by skocdopolet** [reply]
Hello, I have same issue with Symfony 6.2
